frontend/src/screens/setup/ImportMnemonic.tsx

```diff
@@ -90,6 +95,16 @@ export function ImportMnemonic() {
                 Make sure to enter them somewhere safe and private
               </span>
             </div>
+            <div className="flex gap-2 items-center">
+              <div className="shrink-0 text-destructive">
+                <ShieldAlert className="w-6 h-6" />
+              </div>
+              <span className="text-destructive">
+                Your recovery phrase <b>cannot</b> restore funds in lightning
+                channels. Please contact Alby support before proceeding if you
+                had channels on a different device.
+              </span>
+            </div>
```

Comment:
I really like those checkboxes above which re-confirm the warnings. Should we add a similar check box here to tell the lightning channel balance is not restored?

(My understanding is that if the user ignores the warnings and continues then they might not be able to recover the lightning balance(?) If yes then it's good to add one here too)

Reply:
Good idea, thanks